code-samples/eventing/helloworld/helloworld-go/README.md

```diff
@@ -360,85 +362,87 @@ Play around with the CloudEvent attributes in the curl command and the trigger s
 `source=knative/eventing/samples/hello-world`. This event enters the eventing
 mesh via the Broker and can be delivered to other services using a Trigger
 
-1. Using the following example, create a YAML file for a Pod that receives any
-CloudEvent and logs the event to its output:
-
-      ```yaml
-      # event-display app deployment
-      apiVersion: apps/v1
-      kind: Deployment
-      metadata:
-        name: event-display
-        namespace: knative-samples
-      spec:
-        replicas: 1
-        selector:
-          matchLabels: &labels
+1.  Using the following example, create a YAML file for a Pod that receives any
+    CloudEvent and logs the event to its output:
+
+        ```yaml
```

Comment:
the indent on this one is incorrect: https://github.com/knative/docs/blob/9ea2b2fc6488979243217cafcda96ec1c875bc75/code-samples/eventing/helloworld/helloworld-go/README.md#verify-reply-from-helloworld-go-app